ERCS/erc-XXXX.md

```diff
@@ -0,0 +1,244 @@
+---
+eip: XXXX
+title: Smart Account Interfaces
+description: A simple way for apps, wallets, and smart account tooling to interact with Smart Accounts
+author: Derek Chiang (@derekchiang), Fil Makarov (@filmakarov), Garvit Khatri (@plusminushalf), Kristof Gazso (@kristofgazso), Derek Rein (@arein), Juan Caballero (@bumblefudge), Tomas Rocchi (@tomiir), Jake Moxey (@jxom)
```

Comment:
I don't think I had much of an impact here – so happy to omit my name to weight the credit on others! 😄